editoast/src/models/pathfinding.rs

```diff
@@ -33,15 +33,14 @@ use serde::{Deserialize, Serialize};
     Insertable,
 )]
 #[derivative(Default(new = "true"))]
-#[model(table = "osrd_infra_pathmodel")]
+#[model(table = "pathfinding")]
```

Comment:
How about this ? 
Or maybe `pathfinding_result`
```suggestion
#[model(table = "infra_path")]
```

Reply:
will change this name later when refactoring the path model